src/apps/webpack.shared.js

```diff
@@ -23,12 +22,19 @@ module.exports = {
         filename: `${filename}.html`,
         template: path.join(__dirname, '../index.html'),
       }),
+      new webpack.DefinePlugin({
+        'process.env.PLAIN_HMR': false,
```

Comment:
Why do we need to do this?

Reply:
https://stackoverflow.com/questions/41359504/webpack-bundle-js-uncaught-referenceerror-process-is-not-defined

Reply:
Did not find HRM mentioned in that thread :D

Reply:
We have to use this to expose the process.env.PLAIN_HMR. 

Apparently you cant just use `process.env.*` now, so this is a way to provide a default value (false) in case the env variable isnt defined.

Reply:
otherwise you'll get an error - process is undefined.

Reply:
We don't do that here but there is no error https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/chat/webpack.config.js

Reply:
Yeah because they never use process.env in the app